web/templates/note/index.html.eex

```diff
@@ -1,9 +1,11 @@
 <h1>Listing Notes</h1>
 
+<p><%= link "New note", to: note_path(@conn, :new) %></p>
+
 <ul>
   <%= for note <- @notes do %>
     <li>
-      <div><b><%= note.title %></b></div>
+    <div><b><%= link note.title, to: note_path(@conn, :show, note.id) %></b> (<%= note.id %>)</div>
```

Comment:
Will break up the beefy line there. The id is a debugging aid for me at the moment. I know which id exists and which ones were deleted. Is that cool?